server/internal/store/user_org.go

```diff
@@ -9,3 +9,31 @@ type UserOrganization struct {
 	User         uint `gorm:"uniqueIndex:user_org"`
 	Organization uint `gorm:"uniqueIndex:user_org"`
 }
+
+// CreateUserOrganization adds a user to an organization.
+func (s *S) CreateUserOrganization(orgID, userID string) (*UserOrganization, error) {
+	org, err := s.GetOrganization(orgID)
```

Comment:
Could you check if the org belongs to the tenant of the request? We could move this check to the caller.